extensions/blocks/calendly/calendly.php

```diff
@@ -63,7 +63,8 @@ function load_assets( $attr, $content ) {
 	$text_color              = get_attribute( $attr, 'textColor' );
 	$primary_color           = get_attribute( $attr, 'primaryColor' );
 	$classes                 = Blocks::classes( FEATURE_NAME, $attr, array( 'calendly-style-' . $style ) );
-	$block_id                = wp_unique_id( 'calendly-block-' );
+	$block_id                = wp_unique_id();
+	$html_id                 = 'calendly-block-' . $block_id;
```

Comment:
A minor one. Not sure about this var name since it seems to be kind of ambiguous?
Maybe something like `calendly_html_attr_id` or just `calendly_attr_id`? Please feel free to ignore it, too.